--- setup.py	2023-10-05 22:16:49.000000000 +0200
+++ setup.py.new	2024-07-15 21:15:57.034407145 +0200
@@ -64,7 +64,8 @@
             define_macros += [('MTMETIS_64BIT_PARTITIONS', None)]
             libraries += ['mtmetis', 'wildriver']
 
-        extra_compile_args = {'cxx': ['-O3']}
+        # extra_compile_args = {'cxx': ['-O3']}
+        extra_compile_args = {'cxx': ['-O3', '-DGLOG_USE_GLOG_EXPORT']}
         if not os.name == 'nt':  # Not on Windows:
             extra_compile_args['cxx'] += ['-Wno-sign-compare']
         if sys.platform == 'darwin':  # On macOS:
@@ -91,7 +92,7 @@
             define_macros += [('WITH_CUDA', None)]
             nvcc_flags = os.getenv('NVCC_FLAGS', '')
             nvcc_flags = [] if nvcc_flags == '' else nvcc_flags.split(' ')
-            nvcc_flags += ['-O3']
+            nvcc_flags += ['-O3', '-DGLOG_USE_GLOG_EXPORT']
             if torch.version.hip:
                 # USE_ROCM was added to later versions of PyTorch
                 # Define here to support older PyTorch versions as well:
