--- setup.py	2023-10-05 20:36:18.000000000 +0200
+++ setup.py.new	2024-07-15 22:06:58.553935150 +0200
@@ -44,7 +44,7 @@
         if sys.platform == 'win32':
             define_macros += [('torchscatter_EXPORTS', None)]
 
-        extra_compile_args = {'cxx': ['-O3']}
+        extra_compile_args = {'cxx': ['-O3', '-DGLOG_USE_GLOG_EXPORT']}
         if not os.name == 'nt':  # Not on Windows:
             extra_compile_args['cxx'] += ['-Wno-sign-compare']
         extra_link_args = [] if WITH_SYMBOLS else ['-s']
@@ -69,7 +69,7 @@
             define_macros += [('WITH_CUDA', None)]
             nvcc_flags = os.getenv('NVCC_FLAGS', '')
             nvcc_flags = [] if nvcc_flags == '' else nvcc_flags.split(' ')
-            nvcc_flags += ['-O3']
+            nvcc_flags += ['-O3', '-DGLOG_USE_GLOG_EXPORT']
             if torch.version.hip:
                 # USE_ROCM was added to later versions of PyTorch.
                 # Define here to support older PyTorch versions as well:
