diff --unified --recursive --text package.orig/ign-math-ignition-math6_6.4.0/src/RollingMean.cc package.new/ign-math-ignition-math6_6.4.0/src/RollingMean.cc
--- package.orig/ign-math-ignition-math6_6.4.0/src/RollingMean.cc	2020-05-20 18:06:52.306672954 +0200
+++ package.new/ign-math-ignition-math6_6.4.0/src/RollingMean.cc	2020-05-20 18:07:21.856674316 +0200
@@ -17,6 +17,7 @@
 
 #include <numeric>
 #include <deque>
+#include <limits>
 #include "ignition/math/RollingMean.hh"
 
 using namespace ignition::math;
